foundation/foundation-mda/src/main/resources/templates/data-delivery-data-records/data-delivery-combined-data-records-java.pom.xml.vm

```diff
@@ -31,5 +31,10 @@
             <groupId>com.boozallen.aissemble</groupId>
             <artifactId>extensions-data-delivery-spark</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye.reactive</groupId>
+            <artifactId>smallrye-reactive-messaging-kafka</artifactId>
```

Comment:
I: Key fix. Otherwise, the serde classes will be generated, but not have access to the Kafka classes that they use.  We could _conditionally_ add this dependency that's tied to the same condition that triggers the serde generation.  However:
 - it's not unlikely that the condition will be added after initial generation (requires updating a pipeline messaging step to use the record for inbound/outbound)
 - the fix isn't obvious if you do add record messaging
 - a single extra dependency is relatively cheap
 - it's very easy to detect and remove an unused dependency